README.md

```diff
@@ -10,18 +10,40 @@ The humble beginnings of a Nim library similar to web3.[js|py]
 ## Installation
 
 You can install the developement version of the library through nimble with the following command
+
 ```
 nimble install https://github.com/status-im/nim-web3@#master
 ```
 
+## Development
+
+You should first run `./simulator.sh` which runs `ganache-cli`
+
+This creates a local simulated Ethereum network on your local machine and the tests will use this for their E2E processing
+
+### Interaction with nimbus-eth2
+
+This repo relies heavily on parts of the `nimbus-eth2` repo.
+In order to work properly here you need to `source /nimbus-eth2/env.sh`
+
+#### Example
+
+Need to log the output of the `websocketclient.nim` responses:
```

Comment:
This also feels like a rather arbitrary tip that doesn't belong to the README of a library like nim-web3 which has quite large API surface area which is appropriate to cover here.

Maybe you can start a separate contributors guide, which is linked from the README and starts with a debugging tips and tricks section, but with just this tip alone, it would feel quite empty.

Reply:
Ok that seems like a good idea, I can start writing something in a different folder and perhaps link to it a little later.

However this tooltip would have been exceedingly helpful for me while starting this issue as I had no idea 😵‍💫

Reply:
Do you suggest we start a new repo for this comment or I should make a PR to the status style guide?